test/__mocks__/validation.mock.ts

```diff
@@ -105,7 +105,23 @@ export const ENTITY_WITH_MAX_LENGTH_FIELDS_MOCK = {
   ...PRINCIPAL_ADDRESS_WITH_MAX_LENGTH_FIELDS_MOCK
 };
 
-export const ENTITY_WITH_INVALID_CHARACTERS_FIELDS_MOCK = {
+export const REGISTER_ENTITY_WITH_INVALID_CHARACTERS_FIELDS_MOCK = {
+  entity_name: "^ name of a company ^",
```

Comment:
Should the registration mock have no entity_name?

Reply:
Whoops, good spot!